java-checks/src/main/java/org/sonar/java/checks/ProtectedMemberInFinalClassCheck.java

```diff
@@ -47,29 +52,39 @@
   public void visitNode(Tree tree) {
     ClassTree classTree = (ClassTree) tree;
     if (ModifiersUtils.hasModifier(classTree.modifiers(), Modifier.FINAL)) {
-      for (Tree member : classTree.members()) {
-        checkMember(member);
-      }
+      classTree.members().forEach(this::checkMember);
     }
   }
 
   private void checkMember(Tree member) {
```

Comment:
FP. will be handled by resolution of [SONARJAVA-958](https://jira.sonarsource.com/browse/SONARJAVA-958)